apache/trafficcontrol
# Bugfix Traffic Portal UI

## What does this PR (Pull Request) do?


- [x] This PR fixes a bug in Traffic_Portal's UI end-end testing.  https://github.com/apache/trafficcontrol/pull/4908/files/e71e4255fc0c429ecdca0ffd3d1acd640abac4da#diff-97233245722ee5dad1b7499a628647ed
There isn't an issue number

- [x] It also updated the password in Traffic Portal's end-to-end/config.json file


## Which Traffic Control components are affected by this PR?


- Traffic Portal

## What is the best way to verify this PR?

1. Run CIAB
2. Run Traffic_Portal locally (make sure the Traffic_Ops url point to CIAB traffic ops url)
3. Ensure protractor and webdriver-manage are installed.
4. Run `webdriver-manager start`
5. Run `protractor config.js` from traffic_control/traffic_portal/test/end-to-end/ folder
6. All 72 specs should pass. `72 specs, 0 failures`

## If this is a bug fix, what versions of Traffic Control are affected?


- master (5605d22)

## The following criteria are ALL met by this PR


- [x] This does not include a database migration


## Additional Information
